.github/workflows/nodejs.yml

```diff
@@ -0,0 +1,95 @@
+name: webpack-dev-middleware
+
+on:
+  push:
+    branches:
+      - master
+      - next
+  pull_request:
+    branches:
+      - master
+      - next
+
+jobs:
+  #  lint:
```

Comment:
Please uncomment lint job if possible

Reply:
@hiroppy I will do it after we will fix the problem (before merge)

Reply:
okay, thanks